content/english/Kusto-KC7/activity-2.md

```diff
@@ -0,0 +1,213 @@
+- Geographical Pattern of Attack - Are organizations in a particular country predominantly being targeted? Transversely, is an entity targeting organizations all over the world EXCEPT one country....potentially their own?
+
+The difficulty level of cyber attribution makes it an attractive method of attack for groups with the knowledge, resources, and motivation to attempt it while hiding traces of their involvement. With that in mind, our mission is to help raise the next generation of Cyber Defenders to combat cyber attacks. As difficult as it can be, this type of work has real world impact. Check out some examples of success stories below.
+- Exposing POLONIUM activity and infrastructure targeting Israeli organizations: https://vulners.com/mssecure/MSSECURE:A2F131E46442125176E4853C860A816C 
+
+### Classifications of Adversaries
+
+There are 3 main categories we use to classify adversaries: **hacktavists, cyber criminals, and nation sponsored actors**.
+
+Hacktavists are people who aren't affiliated with a specific government and they're not in it for money typically. They believe passionately in some sort of ideal and use cyber attacks to further their mission.
+Key Traits
+- Hacking to spread idealism
+- Seek political or social change
+- Not necessarily the most technically advanced
+- E.g. Cult of the dead cow / Anonymous
+
+Cyber criminals are hacking with the main motivation of financial gain.
+Key Traits
+- Motivated by money
+- Indiscriminate targeting
+- Ransomware / Business Scams / Hack and Leak
+- Level of organization varies 
+- Lone wolves & Organized mafias 
+
+Nation Sponsored are actors operating on behalf of their government. Many governments financially support and direct cyber groups to hack in line with the government's goals which could be political, financial, defense related, etc.
+- Government sponsored or affiliated
+- Highly funded
+- Highly motivated
+- Selective in targeting
+- Long term outlook
+
+
+
+
+-----------------------------------------------------
+
+
+### It's time to play a game of Whodunit, where we throw out some scenarios and you try to your hand at attribution!
+
+Question 1:
+<img src= "https://github.com/bgrant34/workshops/blob/master/content/english/kusto-kc7/Images/WhodunitQ1.png?raw=true" alt= “WhodunitQ1” width="80%" height="value">
+
+#### Which type of adversary do you think did it?
+
+
+<FORM method="POST"
+      onSubmit="return checkAnswer(this,'C');"
+>
+
+<INPUT TYPE="RADIO" VALUE="A" NAME="cc">
+A. Hacktivist<BR>
+
+<INPUT TYPE="RADIO" VALUE="B" NAME="cc">
+B. Cyber Criminal<BR>
+
+<INPUT TYPE="RADIO" VALUE="C" NAME="cc">
+C. Nation Sponsored<BR>
+
+<INPUT TYPE="SUBMIT" VALUE="Submit Answer">
+
+</FORM>
+
```

Comment:
either use <br/> to add new lines

or make Question 1 -> #### Question 1 etc.

Reply:
I actually added both bc I like how it looks. Thanks